code/workspaces/client-api/src/dataaccess/stackUrlService.js

```diff
@@ -35,10 +35,11 @@ const createJupyterUrl = notebook => token => (token ? `${notebook.url}/tree/?to
 
 const createJupyterlabUrl = notebook => token => (token ? `${notebook.url}/lab?token=${token}` : undefined);
 
-const createRStudioUrl = notebook => tokens =>
-  (tokens ? `${notebook.url}/connect?username=${RSTUDIO_USERNAME}&expires=${tokens.expires}&token=${tokens.token}&csrfToken=${tokens.csrfToken}` : undefined);
+const createRStudioUrl = notebook => tokens => (tokens
+  ? `${notebook.url}/connect?username=${RSTUDIO_USERNAME}&expires=${tokens.expires}&token=${tokens.token}&csrfToken=${tokens.csrfToken}`
+  : undefined);
 
-function requestZeppelinToken(stack, user) {
+function requestZeppelinToken(stack) { // stack, user
```

Comment:
does the `user` need to be removed from this comment?

Reply:
As per my comment above.